pkg/server/sandbox_status_test.go

```diff
@@ -42,9 +42,9 @@ func TestPodSandboxStatus(t *testing.T) {
 		Linux: &runtime.LinuxPodSandboxConfig{
 			SecurityContext: &runtime.LinuxSandboxSecurityContext{
 				NamespaceOptions: &runtime.NamespaceOption{
-					HostNetwork: true,
-					HostPid:     false,
-					HostIpc:     true,
+					Network: runtime.NamespaceMode_POD,
```

Comment:
Why changing the host network setting in the test?

Reply:
The test is to test the convert function `toCRISandboxStatus`. It's just simply convert. :)

I can keep network `NODE`.

Reply:
Ack.

Reply:
Done.